util/keycloak_client.py

```diff
@@ -2,6 +2,7 @@
 from keycloak.admin.users import User, Users
 from keycloak.admin.groups import Groups
 from keycloak.admin.user.usergroup import UserGroups
+from keycloak.admin.user.usergrouproles import UserGroupRoles
```

Comment:
Do we need to bump a dependency somewhere in the setup.py or poetry.lock to pull in this new version of the keycloak client?